cargo-pgrx/src/command/run.rs

```diff
@@ -120,6 +125,10 @@ pub(crate) fn run(
         features,
     )?;
 
+    if install_only {
+        return Ok(());
```

Comment:
Alternatively this early return could be after `start_postgres(pg_config)?;`, I don't have a strong opinion